src/pages/DatasetCatalog.js

```diff
@@ -438,18 +438,18 @@ class DatasetCatalog extends Component {
                               p({ isRendered: property.propertyName !== 'dbGAP' }, [
                                 span({ id: trIndex + "_datasetName", name: "datasetName", isRendered: property.propertyName === 'Dataset Name' }),
                                 span({ id: trIndex + "_datasetId", name: "datasetId", isRendered: property.propertyName === 'Dataset ID' }),
-                                span({ id: trIndex + "_dataType", name: "dataType", isRendered: property.propertyName === 'Data Type' }),
-                                span({ id: trIndex + "_species", name: "species", isRendered: property.propertyName === 'Species' }),
-                                span({ id: trIndex + "_phenotype", name: "phenotype", isRendered: property.propertyName === 'Phenotype/Indication' }),
                                 span({ id: trIndex + "_participants", name: "participants", isRendered: property.propertyName === '# of participants' }),
+                                span({ id: trIndex + "_phenotype", name: "phenotype", isRendered: property.propertyName === 'Phenotype/Indication' }),
                                 span({ id: trIndex + "_description", name: "description", isRendered: property.propertyName === 'Description' }),
+                                span({ id: trIndex + "_dataType", name: "dataType", isRendered: property.propertyName === 'Data Type' }),
+                                span({ id: trIndex + "_species", name: "species", isRendered: property.propertyName === 'Species' }),
```

Comment:
I don't think this is working as expected. Some of the columns in this table are hard-coded, so those are easy. Those are:
* Dataset ID
* Consent ID
* SC-ID
* Structured DUL
* Action buttons

Everything between Dataset ID and Consent ID is dynamically rendered on line 369. It depends on the display order of `this.state.dataSetList.dictionary` which comes from the server and is not part of the UI. We'll need to either disregard the display order and come up with our own, or change this on the server side. My preference (at this point) is to create a custom sort order that we can apply to the dictionary list and use that here instead of going back to the server.

Reply:
Turns out there are two things we need to provide custom sorting on. The `dictionary` controls a portion of the header columns and `catalog.properties` controls those same elements in the row data.